liquibase-dist/src/main/archive/liquibase.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+export LIQUIBASE_HOME=/opt/liquibase
+export PATH=$PATH:$LIQUIBASE_HOME
+source /etc/profile.d/
```

Comment:
The file is installed inside `/etc/profile.d/` . Does it need to source it?

Reply:
Nop